database/reviewsSchema.sql

```diff
@@ -0,0 +1,63 @@
+DROP DATABASE IF EXISTS test;
+
+CREATE DATABASE test;
+
+\c test;
+
+CREATE TABLE reviews (
+  id INTEGER PRIMARY KEY,
+  product_id INTEGER NOT NULL,
+  rating INTEGER NOT NULL,
+  date BIGINT NOT NULL,
+  summary TEXT,
+  body TEXT NOT NULL,
+  recommend BOOLEAN,
+  reported BOOLEAN,
+  reviewer_name TEXT NOT NULL,
+  reviewer_email TEXT NOT NULL,
+  response TEXT,
+  helpfulness INTEGER
+);
+
+COPY reviews FROM 'Users/alishaedington/work/SDC/SDC_files/reviews.csv' delimiter ',' csv header;
+
+CREATE TABLE review_photos (
+  id INTEGER PRIMARY KEY,
+  review_id INTEGER NOT NULL,
+  url TEXT,
+  FOREIGN KEY(review_id) REFERENCES reviews(id)
+);
+
+COPY review_photos FROM 'Users/alishaedington/work/SDC/SDC_files/review_photos.csv' delimiter ',' csv header;
+
+CREATE TABLE characteristics (
+  id INTEGER PRIMARY KEY,
+  product_id INTEGER NOT NULL,
+  name TEXT
+);
+
+COPY characteristics FROM '.Users/alishaedington/work/SDC/SDC_files/characteristics.csv' delimiter ',' csv header;
```

Comment:
You could make path changes to this now, or wait until it actually matters again. I'd probably make a note of it.